Tasks/PackerBuild/DefaultTemplates/default.windows.template.json

```diff
@@ -0,0 +1,66 @@
+{
+  "variables": {
+    "client_id": "",
+    "client_secret": "",
+    "subscription_id": "",
+    "tenant_id": "",
+    "object_id": "",
+    "resource_group": "",
+    "storage_account": "",
+    "image_publisher": "",
+    "image_offer": "",
+    "image_sku": "",
+    "location": "",
+    "script_path": "",
+    "script_name": "",
+    "package_path": "",
+    "package_name": ""    
+  },
+  "builders": [{
+        "type": "azure-arm",
+        "client_id": "{{user `client_id`}}",
+        "client_secret": "{{user `client_secret`}}",
+        "resource_group_name": "{{user `resource_group`}}",
+        "storage_account": "{{user `storage_account`}}",
+        "subscription_id": "{{user `subscription_id`}}",
+        "tenant_id": "{{user `tenant_id`}}",
+        "object_id": "{{user `object_id`}}",
```

Comment:
these will come from SPN. Right now task.json does not have azure endpoint. However, I will add that later and then it will light up.